scripts/backup_db.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+set -euxo pipefail
```

Comment:
Reference for whoever might be interested in the meaning of the flags: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin